lite/backends/nnadapter/nnadapter/driver/imagination_nna/imgdnn_manager.cc

```diff
@@ -298,6 +298,80 @@ imgdnn_tensor ImgdnnManager::CreateFullyConnectedLayer(
   return ConvertQuantTensorType(output_tensor, &quant_param);
 }
 
+imgdnn_tensor ImgdnnManager::CreateMatMulLayer(imgdnn_tensor x_tensor,
+                                               imgdnn_tensor y_tensor,
+                                               imgdnn_quant_param quant_param) {
+  imgdnn_err_code error_code;
+  imgdnn_tensor_descriptor x_desc;
+  IMGDNN_CHECK_SUCCESS(imgdnnGetTensorDescriptor(x_tensor, &x_desc));
+  imgdnn_tensor_descriptor y_desc;
+  IMGDNN_CHECK_SUCCESS(imgdnnGetTensorDescriptor(y_tensor, &y_desc));
+  if (x_desc.dimensions > 2) {
+    NNADAPTER_LOG(FATAL)
+        << "Imgination NNA does not support the dimension of x > 2";
+  }
```

Comment:
NNADAPTER_CHECK_GT(x_desc.dimensions, 2) << "Imgination NNA does not support the dimension of x > 2";

Reply:
done